MPF.md

```diff
@@ -1393,6 +1398,49 @@ especially with public repositories or you may be banned for abuse.
 * Added in 3.15.0, 3.12.3
 * 3.17.0 decreased `package_module_query_installed_ifelapsed` from `60` to `0`
 
+#### For package methods
+
+CFEngine refreshes it's cache of information about packages installed and
+updates available when it evaluates packages promises if the cache has not been
+updated in the number of minutes stored in `package_list_update_ifelapsed` of
+the package method in use. Many package methods in the standard library use the
+value of `default:common_knowledge.list_updates_ifelapsed` for this value which
+can be customized via Augments.
+
+```json
+{
+  "variables": {
+    "default:common_knowledge.list_update_ifelapsed": {
+      "value": "0"
+    }
+  }
+}
+```
+
+**Notes:**
+
+* Unlike *many* variables that can be customized via Augments this variable is
```

Comment:
Why is this different and not in `default:def` scope so that you can use "old style" vars augments?

Reply:
I chose to not put it in `def.default` default because we are not bound to default:def since 3.18 and this was not in a control body. Rather than define a net new variable in def, just to be used by this other variable, I made the actual variable that has been able to retain a setting via Augments instead of over-writing anything that was already defined.

The `vars` key can actually be used to target this in 3.18.0+. 

```json
{
  "vars": {
    "default:common_knowledge.list_update_ifelapsed":  "0"
  }
}
```

And it's fine to use a mix of `vars` and `variables` in `def.json`.

![image](https://github.com/cfengine/masterfiles/assets/202896/a66aafb4-cb4a-4ae2-b709-c5867271e0d3)